integration-tests/src/test_loop/tests/resharding_v3.rs

```diff
@@ -988,11 +1025,13 @@ fn test_resharding_v3_base(params: TestReshardingParameters) {
         let runtime_config_store = RuntimeConfigStore::with_one_config(runtime_config);
         builder = builder.runtime_config_store(runtime_config_store);
     }
+    let num_clients = params.clients.len();
 
-    let TestLoopEnv { mut test_loop, datas: node_datas, tempdir } = builder
+    let mut env = builder
```

Comment:
Why not unpack? It was a bit nicer when unpacked.

Reply:
I also do not like it, but it is required by `create_account` / `delete_account`